app/templates/_bower.json

```diff
@@ -36,24 +36,32 @@
         <%_ if (websocket == 'spring-websocket') { _%>
         "sockjs-client": "1.0.3",
         "stomp-websocket": "2.3.4",
+        <%_ } _%> <%_ if (enableSocialSignIn) { _%>
+        "bootstrap-social": "4.10.1",
         <%_ } _%>
+        "font-awesome": "4.4.0",
```

Comment:
Ok, two options: 
* I remove totally the font-awesome lib, and the icon of the social button too (because icon like Twitter, Facebook dont exist in bootstrap icon package)
* I add font awesome only when social is enabled

Wich one you prefer?